kofta/public/locales/owo/translation.json

```diff
@@ -232,16 +232,16 @@
       "roomDescription": " woom descwiption（＾ｖ＾）"
     },
     "roomStatus": {
-      "fuelingRocket": "Fueling rocket",
-      "takingOff": "Taking off",
-      "inSpace": "In space",
-      "approachingMoon": "Approaching moon",
-      "lunarDoge": "Lunar doge",
-      "approachingSun": "Approaching sun",
-      "solarDoge": "Solar doge",
-      "approachingGalaxy": "Approaching galaxy",
+      "fuelingRocket": "Fuewing rocket uwu",
+      "takingOff": "Tawking off xD",
+      "inSpace": "In space ʕʘ‿ʘʔ",
+      "approachingMoon": "Appwoaching moon  (❁´◡`❁)",
+      "lunarDoge": "Lunar doge (；ω；)",
+      "approachingSun": "Appwoaching sun >_<",
+      "solarDoge": "Solar doge :3",
+      "approachingGalaxy": "Appwoaching galaxy (◠‿◠✿)",
       "galacticDoge": "Galactic Doge",
```

Comment:
Are we sure this string doesn't need translating? 😄

Reply:
oh I forgot that line

Reply:
@Nilstrieb is this pr ready?

Reply:
from my side, yes